src/md/release.md

```diff
@@ -201,6 +252,27 @@ git push apache
 ```
 Merge `@since` tag and change log changes to `master`.
 
+### Documentation
+
+The documentation promotion is done in two steps. First the release documentation component is made available on the website and then the remaining website documentation is deployed.
```

Comment:
I will reword it. The intent is to say that the module documentation is first deployed then the rest of the website is deployed.